parcel_test.go

```diff
@@ -31,57 +34,110 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Тут было все правильно написано, нужно просто убрать if
```suggestion
		require.NoError(t, err)
```